mep3_planning/assets/strategies/ros_demo.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
Ime treba odgovarati upustvu (`robot_strategy_1`)

Reply:
Taj demo fajl cemo verovatno obrisati ili preimenovati u nekom trenutku, pa sam stavio kao placeholder ime fajla. Svakako necemo imati samo jednu strategiju koja se zove `robot_strategy_1` ili `ros_demo`.

Reply:
Jasno, samo bi bilo dobro da sve out-of-the-box. Za nove clanove, mep3 je previse novog i treba maksimalno da im olaksamo :)